src/elements/fields/PaymentMethodField.jsx

```diff
@@ -0,0 +1,194 @@
+    }
+    setCardElement(element.servar.key, elements.getElement(CardElement));
+  }, [stripe, element.servar.key, elements]);
+
+  // In general, mobile styles are supported for the payment method field.  However, we do this using
+  // media queries.  The Stripe card element does not support this so the mobile styles that are set
+  // in media queries directly below are ignored.  This means that mobile styles like color and the various font styles
+  // do not work (mobile settings will not be applied).  However, other mobile styles for things like
+  // background color and borders are set in the parent of the card element and DO WORK.
+  const cardElementOptions = {
+    style: {
+      base: {
+        textDecoration: 'none', // Bug in card element - force the reset to none
+        ...filterKeys(applyStyles.getTarget('field'), supportedStripeCssProps),
+        '::placeholder': {
+          textDecoration: 'none', // Bug in card element - force the reset to none
+          ...filterKeys(
+            applyStyles.getTarget('placeholder'),
+            supportedStripeCssProps
+          )
+        },
+        ':hover': filterKeys(
+          applyStyles.getTarget('hover'),
+          supportedStripeCssProps
+        ),
+        ':focus': filterKeys(
+          applyStyles.getTarget('active'),
+          supportedStripeCssProps
+        )
+      },
+      complete: applyStyles.getTarget('completed'),
+      invalid: {
+        color: ERROR_COLOR,
+        iconColor: ERROR_COLOR
+      }
+    },
+    classes: {
+      base: element.servar.key
+    }
+  };
+  const handleCardChange = (event) => {
+    // Listen for changes in the CardElement
+    // and display any errors as the customer types their card details
+    setFieldError(event.error?.message ?? '');
+    // also set the field data with complete=true so we know
+    // when a card number has been entered and is complete
+    onChange(
+      event.complete
+        ? { complete: true, card_data: { brand: event.brand } }
+        : ''
+    );
+  };
+
+  return (
+    <div
+      css={{
+        maxWidth: '100%',
+        position: 'relative',
+        pointerEvents: editable ? 'none' : 'auto',
+        ...applyStyles.getTarget('fc')
+      }}
+      {...elementProps}
+    >
+      {fieldLabel}
+      <div
+        css={{
+          position: 'relative',
+          width: '100%',
+          whiteSpace: 'nowrap',
+          overflowX: 'hidden',
+          padding: '0 2px',
+          display: 'flex',
+          alignItems: 'center',
+          ...applyStyles.getTarget('sub-fc'),
+          '&:focus': applyStyles.getTarget('active'),
+          '&:hover': applyStyles.getTarget('hover'),
+          ...(inlineError ? { borderColor: ERROR_COLOR } : {})
+        }}
+      >
+        <div css={{ width: '100%', position: 'relative' }}>
+          {/* position an input field under the card element to support html5 errors */}
+          {runtime && errorDisplayMode === 'html5' && (
```

Comment:
editable is set to true in dashboard and falsey in react lib, so you don't need an additional runtime boolean here

Reply:
Ah, I missed editable.  Good to know.  Fixed.